### PR TITLE
remove formats no longer supported

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

File: admin/en/source/pages/upgrade/upgrade.rst [hunk 1]
```diff
@@ -33,6 +33,8 @@ The (optional) parameter <uforge install type> can take the values ``uforge-ee``
 
 .. note:: It is recommended to take a snapshot of all impacted VMs before starting the update procedure, in case you need to rollback.
 
+.. note:: UForge 3.8 fp5 onwards no longer supports Abiquo, Eucalyptus, Flexiant and Nimbula. When upgrading UForge all user images, published images, credential accounts and migrations using these formats will be deleted.
```
Comment (cinlloc) [inline admin/en/source/pages/upgrade/upgrade.rst]: I'm not sure this should appear in the doc, we don't have any other example of this from previous removal (cf. Azure Classic EOL). Should the release note be self-sufficient?
Furthermore, what will be the future of such a note when we will be in FP6 or 7 for example, or in 3.9? Will we keep it?

Comment (lqueiroga) [reply admin/en/source/pages/upgrade/upgrade.rst]: Agree with Clement, this note is important for users but I don't think this should appears in the doc but communicate it by another way (release note).

Comment (emuus) [reply admin/en/source/pages/upgrade/upgrade.rst]: @jfknoepfli Do you confirm that adding this note in the release notes is enough? It wouldn't cause too much confusion for the user?

Comment (emuus) [reply admin/en/source/pages/upgrade/upgrade.rst]: ok removed. Will let the dev team add to RN

Comment (cinlloc) [reply admin/en/source/pages/upgrade/upgrade.rst]: Seems the note is still here. Maybe forgot to `git add` this file?

**Comment by emuus** [comment]
sorry, now removed both notes

**Comment by lqueiroga** [comment]
Thanks for this PR

**Pull request merged**